app/decorators/mfa_context.rb

```diff
@@ -18,9 +18,20 @@ def phone_configuration(id = nil)
     phone_configurations.find { |cfg| cfg.id.to_s == id.to_s }
   end
 
+  # TODO: Maybe this should be renamed to reflect that it is a roaming authenticator?
+  # Potentially the MFA context does not need to differentiate and can leave that to the presenters
+  # similar to how phone has voice and SMS modes
```

Comment:
My first thought was to rename this as well.

I think the only case we'd want to differentiate between roaming/platform is if we require that accounts also add a non-platform MFA to their accounts as well

Reply:
Yeah, the more I think about it the more I think if should just be `webuathn_configurations`.

Reply:
Okay, yanked it